whois-query/src/test/java/net/ripe/db/whois/query/integration/ProxyProtocolTestIntegration.java

```diff
@@ -0,0 +1,122 @@
+package net.ripe.db.whois.query.integration;
+
+import com.amazonaws.proprot.Header;
+import com.amazonaws.proprot.ProxyProtocol;
+import com.amazonaws.proprot.ProxyProtocolSpec.AddressFamily;
+import com.amazonaws.proprot.ProxyProtocolSpec.Command;
+import com.amazonaws.proprot.ProxyProtocolSpec.TransportProtocol;
+import com.google.common.collect.Lists;
+import net.ripe.db.whois.common.IntegrationTest;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.query.QueryServer;
+import net.ripe.db.whois.query.support.AbstractQueryIntegrationTest;
+import net.ripe.db.whois.query.support.TestPersonalObjectAccounting;
+import org.apache.commons.io.IOUtils;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.Inet4Address;
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.nio.charset.StandardCharsets;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@Category(IntegrationTest.class)
+public class ProxyProtocolTestIntegration extends AbstractQueryIntegrationTest {
```

Comment:
I'll suggest renaming to `WhoisProxyProtocolTestIntegration.java` to keep a symmetry with `NrtmProxyProtocolTestIntegration.java`

Reply:
Yep will do.

Reply:
Done.